app/store/templates/order-history.html

```diff
@@ -0,0 +1,21 @@
+<ion-view view-title="Order History">
+  <!-- do you want padding? -->
+  <ion-content class="padding">
+    <!-- content goes here -->
+    <ion-list>
+      <a class="item item-thumbnail-left" href="#">
+        <img src="http://res.cloudinary.com/duqhan/image/upload/v1484909270/1_h7vdx5.jpg">
+        <h2>Moto E3 Power (Black,16 GB)</h2>
```

Comment:
I'm assuming all of this will be dynamic instead of static content in the future after integrating with the backend?